modules/openapi-generator/src/main/resources/dart/libraries/dio/api.mustache

```diff
@@ -116,7 +116,7 @@ class {{classname}} {
     );
     {{#returnType}}
 
-    {{{returnType}}} _responseData;
+    {{{.}}}? _responseData;
```

Comment:
Don't we know if the response may be null? Maybe we can also change the operation generic return type?

Reply:
it won't matter since the final response.data is nullable anyway

Reply:
True, that's a dio probelm.